index.html

```diff
@@ -69,5 +69,130 @@ <h2 class="connect">LET'S CONNECT</h2>
 				</section>
 				<span class="header-shape"></span>
 			</main>
+      <section class="work">
+        <div class="cardWork sec">
+          <img id="snapshot" alt="snapshort img" src="./Scr/images/Snapshoot_.png">
+          <div class="left block_bg">
+            <div class="primary text">
+              <h1 class="project title">Tonic</h1>
+              <div >
+                <ul class="frame">
+                  <li class="client_tag_list"><h2 class="client_canopy">CANOPY</h2></li>
+                  <span class="counter"></span>
+                  <li class="client_tag_role"><h2 class="client_role_text">Back End Dev</h2></li>
+                  <span class="counter_tw"></span>
+                  <li class="client_tag_year"><h2 class="client_yr_text">2015</h2></li>
+                </ul>
+              </div>
+            </div>
+            <div class="client_txt_bd">
+              <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            </div>
+            <ul class="tag_list">
+              <li class="html_tag"><h3 class="html_txt">html</h3></li>
+              <li class="css_tag"><h3 class="css_txt">css</h3></li>
+              <li class="java_tag"><h3 class="java_txt">javaScript</h3></li>
+            </ul>
+            <div id="action">
+              <div class="btn">
+                <button type="button" class="btn_text">See Project</button>
+              </div>               
```

Comment:
- [ OPTIONAL] I think it's not necessary to wrap a button into a div, kindly take it out for semantic purpose :recycle: